src/api/java/com/minecolonies/api/entity/ai/pathfinding/AbstractWalkToProxy.java

```diff
@@ -102,7 +102,7 @@ public boolean walkToBlock(@NotNull final BlockPos target, final int range, fina
                 currentProxy = target;
             }
 
-            this.resetProxyList();
+            proxyList.clear();
             return takeTheDirectPath(target, range, onMove);
```

Comment:
And I thought both things do the same thing =P

Reply:
yea there is a subtle difference, which is that resetProxyList() also clears the currentProxy which just was set on the line before this